tests/token/src/lib.rs

```diff
@@ -0,0 +1,118 @@
+#![no_std]
```

Comment:
This contract doesn't have replay prevention. Was that intentional?

Reply:
Woops, that wasn't meant to be part of this PR. But yeah it doesn't have replay, its just a hack I'm using to play with apis.